api/test/robot_tests/web_on_chain/web_on_chain.robot

```diff
@@ -10,16 +10,14 @@ Resource            ../variables.resource
 Resource            ../front_ends/variables.resource
 Resource            ../front_ends/keywords.resource
 
-Suite Setup         Suite Setup
+Suite Setup         Set Suite Variables
```

Comment:
Tests are probably failing due to this change. The keyword `Set Suite Variables` should be inside the `keywords.resource` file and this Keyword should be called in the `Suite Setup` keyword.